Automattic/jetpack
# VideoPress: Add settings to the React interface

#### Changes proposed in this Pull Request:

Bring VideoPress settings over to the React interface. I think this is pretty solid so far but would love @dereksmart's input on the React code, as well as some experienced eyes on the API bits. This PR should be considered in-progress for now. 

![screen shot 2016-09-30 at 2 49 08 pm](https://cloud.githubusercontent.com/assets/5528445/19004517/42114394-8724-11e6-84f8-145ec1b22dc3.png)

![screen shot 2016-09-30 at 2 51 48 pm](https://cloud.githubusercontent.com/assets/5528445/19004520/463c8c58-8724-11e6-9e23-6c605473669d.png)
#### Testing instructions:
- Spin up a new Jetpack site and activate VideoPress. Follow the rabbit hold within VideoPress settings to create and connect a new WordPress.com Premium account to activate VideoPress. Ensure you can upload and embed videos as usual. Ensure all VideoPress settings save and reflect properly.
- Apply this PR to an existing VideoPress site and ensure you can upload and embed videos as usual. Ensure all VideoPress settings save and reflect properly.
- Attempt to alter settings and/or use VideoPress features as an Author and ensure that settings prevent unauthorized access where configured to do so.


#### Proposed changelog entry for your changes:
---
